src/common/logger.ts

```diff
@@ -17,8 +17,11 @@
  * ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
+import dotenv from 'dotenv';
 import { type Logger, type LoggerOptions, pino } from 'pino';
 
+dotenv.config();
+
```

Comment:
not a request for change, just dropping this here to document there's a "cleaner" alternative:
```suggestion
import { type Logger, type LoggerOptions, pino } from 'pino';

import 'dotenv/config';

```

Reply:
made this change as suggested. thanks for offering a cleaner alternative.